lib/Doctrine/Common/DataFixtures/Purger/ORMPurger.php

```diff
@@ -43,6 +43,9 @@ class ORMPurger implements PurgerInterface, ORMPurgerInterface
      */
     private $excluded;
 
+    /** @var string[]|null */
```

Comment:
should be `list<string>|null` to be more precise.

Reply:
Done, but will be great if you describe the difference. Also it could set `array<string>`

Reply:
`string[]` (or `array<string>` which is the same but with a syntax supporting union types for the type of values) is only specifying the type of values. It is technically the same than `array<int|string, string>`.

`list<string>` is a lot more strict. It forces to have a list, i.e. sequential integer keys (Psalm has full support for lists. PHPStan does not yet have it and treats `list<string>` as an alias of `array<int, string>`, so not reporting cases where keys are not sequential)